README.md

```diff
@@ -31,6 +31,59 @@ LEMMY_BB_BACKEND=https://example.com cargo run
 
 Then open [127.0.0.1:1244](http://127.0.0.1:1244) in your browser. You can login with your existing account.
 
+### Standalone
+
+Follow these steps to install only the lemmyBB frontend on your server.
+
+First, ssh into your server and prepare by cloning the code repository.
+```
+cd /opt
+git clone https://github.com/LemmyNet/lemmyBB.git
+```
+
+Change to the folder, create lemmybb binary and create a link of it to the root folder
+
+```
+cd lemmyBB
+cargo build --release
+ls /opt/lemmyBB/target/debug/lemmy_bb /opt/lemmyBB/lemmy_bb
```

Comment:
Should probably be mv instead of ls. Also should be release and not debug.

Could also leave this in the same location, and instead set working directory to `/opt/lemmyBB/lemmy_bb` in systemd unit file.

Reply:
i do not understand the second paragraph

Reply:
It should work with this in systemd unit file:
```
WorkingDirectory=/opt/lemmyBB/
ExecStart=/opt/lemmyBB/target/release/lemmy_bb
```